### PR TITLE
JENKINS-69301 Update handlebars and replace js-libs with parcel

### PR DESCRIPTION
**Pull request by jgreffe**
https://issues.jenkins.io/browse/JENKINS-69301
Upgrade `handlebars` to 4.7.7 as a self-contained JS
Remove `jenkins-js-libs` and `jenkins-js-builder` dependencies
Move from `jenkins-js-modules` to `@jenkins-cd/js-modules`
Use `parcel` 2 as packager instead of gulp
Still use adjunct and js-modules for JS extension points

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by jgreffe** [comment]
Not sure why there's this error during build:
```
[2022-08-19T13:53:12.412Z] [INFO] > parcel build --public-url ./ ./src/main/js/stageview.js --dist-dir ./src/main/webapp/jsmodules && parcel build --public-url ./ ./src/main/js/stageview_adjunct.js --dist-dir ./target/generated-adjuncts/org/jenkinsci/pipeline
[2022-08-19T13:53:12.412Z] [INFO] 
[2022-08-19T13:53:12.412Z] [INFO] Error: The specified module could not be found.
[2022-08-19T13:53:12.412Z] [INFO] \\?\C:\Jenkins\workspace\ipeline-stage-view-plugin_PR-240\ui\node_modules\@parcel\source-map\parcel_sourcemap_node\artifacts\index.win32-x64-msvc.node
```

**Review by scherler — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-buildPlugin(useContainerAgent: true, configurations: [
```
Comment (jgreffe) [inline Jenkinsfile:1]: Temporary configuration in respect to https://github.com/jenkins-infra/helpdesk/issues/3105

File: Jenkinsfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+buildPlugin(useContainerAgent: false, configurations: [
```
Comment (jgreffe) [inline Jenkinsfile:1]: Temporary configuration in respect to https://github.com/jenkins-infra/helpdesk/issues/3105

File: pom.xml [hunk 3]
```diff
@@ -61,9 +61,9 @@
         <bom>2.289.x</bom>
         <jenkins.version>2.289.1</jenkins.version>
         <changelist>-SNAPSHOT</changelist>
-        <node.version>10.17.0</node.version>
-        <npm.version>6.14.8</npm.version>
-        <frontend-version>1.10.0</frontend-version>
+        <node.version>18.7.0</node.version>
+        <npm.version>8.15.0</npm.version>
+        <frontend-version>1.12.1</frontend-version>
```
Comment (jgreffe) [inline pom.xml:66]: Upgrade in order to make it pass with Mac M1 chips: https://github.com/eirslett/frontend-maven-plugin/issues/952

**Comment by basil** [comment]
Does this still need to be released?

**Pull request merged**